pkg/controller/persistentvolume/types.go

```diff
@@ -26,10 +26,16 @@ import (
 )
 
 const (
+	// A PVClaim can request a quality of service tier by adding this annotation.  The value of the annotation
+	// is arbitrary.  The values are pre-defined by a cluster admin and known to users when requesting a QoS.
+	// For example tiers might be gold, silver, and tin and the admin configures what that means for each volume plugin that can provision a volume.
+	qosProvisioningKey = "volume.experimental.kubernetes.io/quality-of-service"
+	// When a PVClaim requests a QoS tier, the "provisionable" annotation is applied so that the provisioner will process the request for a new volume.
+	provisionableKey = "volume.experimental.kubernetes.io/provisionable"
```

Comment:
Because the official bind is now pv.Spec.ClaimRef, I was able to remove this annotation entirely and all its usages.